source/manual/migrate-to-dart-sass-from-libsass.html.md

```diff
@@ -61,6 +61,8 @@ Remove references to CSS files from the Sprockets manifest file: `app/assets/con
   ...
 ```
 
+**Note**, do not delete references to vendor CSS files, stylesheets placed in the `vendor/assets` folder, to ensure they are processed by Sprockets.
```

Comment:
I think the part in commas should be in brackets, otherwise it reads a bit oddly.